docs/topics/technical-operational/index.md

```diff
@@ -78,10 +83,11 @@ Engage in FEGA discussions happening with existing and interested FEGA nodes wit
 
 ## 3. Evaluate your implementation
 
-- Assess the technical and operational maturation of your node by doing a self-assessment against the [Federated EGA Maturity Model](https://inab.github.io/fega-mm/)
-- Determine compliance of services with FEGA specifications by performing [compliance tests](TBD)
-- Evaluate ability to scale services by performing [stress tests](TBD)
-- Demonstrate the full set of FEGA node services for users by planning your [FEGA end-to-end demonstrator](https://docs.google.com/document/d/1m7WDC112e73Kw79baZcsRsQOkAAGKtp_AiqJRhrgtUk/edit?usp=sharing)
+- Understand the domains in which a node matures using the [Federated EGA Maturity Model](https://ega-archive.github.io/FEGA-onboarding/topics/maturity-model/)
+- Assess the technical and operational maturation of your node by doing a **self-assessment against the Federated EGA Maturity Model** (Coming soon!)
```

Comment:
Is it still coming soon? Didn't Albert show it during our meeting at BSC?

Reply:
I'm not sure if the version of the self-assessment tool that was provided to the inaugural nodes is the final version that will be shared with anyone. I'll leave it to Albert et al. to update this point at a later date.